config/traits.yml

```diff
@@ -489,14 +489,14 @@ traits:
       label:  Canopy light environment required for reproduction
       allowed_values_min: 0
       allowed_values_max: 8
-    upper_cuticle_thickness:
+    cuticle_thickness_adaxial:
```

Comment:
should this start with `leaf_`?

Reply:
Similar for cuticle_thickness_abaxial, epidermis_thickness_adaxial, fluorescence_*, hypodermis_*? Or are we treating epidermis, hypodermic etc as the structure?

Reply:
There are quite a few more like this - I was wondering the same. I won't merge yet and we can discuss.